samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AssessmentSettingsBean.java

```diff
@@ -203,6 +203,8 @@
   private ExtendedTime extendedTime;
   private ExtendedTime transitoryExtendedTime;
   private boolean editingExtendedTime = false;
+
+  private int activeTabIndex = 0;
```

Comment:
Can we make these values named constants? It'll be easier to understand the code in the future when this PR isn't so fresh in our minds. Magic numbers are a little ambiguous when reading out of context: "What does `0` represent? What does `2` represent?", etc.

Also if you put a Lombok `@Getter` and `@Setter` on this, then you don't have to manually write the getter and setter below.